apps/ensadmin/src/components/identity/index.tsx

```diff
@@ -39,32 +39,19 @@ export function Identity({
     setMounted(true);
   }, []);
 
-  // if the ENS namespace is the ens-test-env, always show the truncated address and not look up the primary name.
-  if (namespaceId === ENSNamespaceIds.EnsTestEnv) {
-    // TODO: come back to this later only when ENSNode exposes its own APIs for primary name lookups
-
-    return <AddressDisplay address={address} namespaceId={namespaceId} />;
-  }
-
-  const ensRootChainId = getENSRootChainId(namespaceId);
-
-  // Lookup the primary name for address using wagmi
-  const {
-    data: ensName,
-    isLoading,
-    isError,
-  }: UseEnsNameReturnType<string> = useEnsName({
+  // Lookup the primary name for address using ENSNode
+  const { data, status } = usePrimaryName({
     address,
-    chainId: ensRootChainId,
+    chainId: 1,
```

Comment:
What if we need to display data from Sepolia? I reckon we should still use the `ensRootChainId` as before. Please let me know what you think, @shrugs?